saliensHack.user.js

```diff
@@ -48,12 +48,12 @@
  * SOFTWARE.
  */
 
-(function() {	
+(function() {
     if (typeof unsafeWindow !== "undefined")
-    	unsafeWindow.requestAnimationFrame = c => { setTimeout(c, 1000 / 60); };
+        unsafeWindow.requestAnimationFrame = c => { setTimeout(c, 1000 / 60); };
 
     // Game broke reload and try again
-    GameLoadError = function() {
+    var GameLoadError = function() {
```

Comment:
GameLoadError is actually a variable created by the game. It is the function that handles the error modal that displays and clicking okay reloads the page. This is just overwriting that variable to go ahead and reload the page

Reply:
got it, removing that